aldryn_config.py

```diff
@@ -93,6 +111,12 @@ def to_settings(self, data, settings):
 
         settings['CMS_PERMISSION'] = data['permissions_enabled']
 
+        if data['cms_content_cache_duration']:
+            settings['CMS_CACHE_DURATIONS']['content'] = data['cms_content_cache_duration']
```

Comment:
is `settings['CMS_CACHE_DURATIONS']` already defined?

Just add `settings.setdefault('CMS_CACHE_DURATIONS', {})` above the `if` if it is not.